assets/css/edd-admin.css

```diff
@@ -2431,6 +2420,92 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div#post-body-content {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: 1fr;
+	grid-gap: 20px;
+	max-width: 1000px;
+}
+
+.postbox [class^="edd-export"] {
+	grid-template-rows: 1fr;
+	max-width: 400px;
+}
+
+.edd-reports-wrapper .postbox {
+	margin-bottom: 0;
+}
+#edd-export-earnings select.edd-select {
+	width: 49%;
+}
+
+span.edd-from-to-wrapper {
+	margin-bottom: 15px;
+}
+
+.edd-export-sales-earnings {
+	grid-template-columns: 1fr;
+}
+
+.edd-from-to-wrapper {
+	margin-right: 0;
+}
+
+#edd-order-export-start,
+#edd-order-export-end,
+#edd-orders-export-start,
+#edd-orders-export-end,
+#edd-taxed-orders-export-start,
+#edd-taxed-orders-export-end,
+#edd-api-requests-export-start,
+#edd-api-requests-export-end,
+#edd-file-download-export-start,
+#edd-file-download-export-end {
+	width: 50%;
+	margin-right: 0px;
+}
+
+.edd-import-export-form {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+}
+
+div.to-and-from-container {
```

Comment:
I'm not a fan of setting specific heights on elements because I feel like it's spackling over things we should fix instead. If you set this element to use CSS Grid, something like:

```css
div.to-and-from-container {
    display: grid;
    grid-template-columns: 1fr 1fr;
    grid-gap: 8px;
    margin-bottom: 16px;
}
```

(This may need to be more specific for the exports context actually.) Once you have that, though, you can remove the negative top margin on the `#to-label`, and get more appropriate padding or margins set.

Reply:
Finishing reading the code and I am guessing the 49% on the selects was the best option with the original markup, but I think your markup change was a great idea and will allow more flexible CSS.

Reply:
@robincornett Take a look at this and let me know what you think.